configuration.adoc

```diff
@@ -1803,6 +1810,101 @@ Example, "userSMIMECertificate, UserCertificate"
 
 To set up another external LDAP server, click *Add Configuration*.
 
+=== Setting up server based S/MIME feature
```

Comment:
Suggest:
Setting up for using the S/MIME feature using the server based solution.

Also recommend having this section first, as I suspect this is the preferred method?